src/Components/App.js

```diff
@@ -5,15 +5,17 @@ import Home from './Home';
 import Pet from './Pet';
 import PetApplications from './PetApplications';
 import Error from './Error';
-
-
+import ApplicationFormModal from './ApplicationFormModal';
+import DonationFormModal from './DonationFormModal';
 
 function App() {
   return (
     <div className="App">
       <Routes>
         <Route path="/"  element={<Home />} />
         <Route path="pet/:id" element={<Pet />} /> 
+        <Route path='application' element={<ApplicationFormModal />} />
```

Comment:
We will have to be mindful with these 2 routes when we link them to the appropriate pet_id